fastlane/Fastfile

```diff
@@ -64,38 +64,6 @@ platform :ios do
       )
     end
 
-
-  desc "Copies the branding assets from the themes repository to the ios-app repository"
-    lane :prepare_branding_assets do |options|
-
-    if Dir.exist?("../../customer-themes-tmp") == false
-		UI.user_error!("Themes repository path does not exists!")
-    end
-
-    sh "git -C ../../customer-themes-tmp pull"
-
-    path = ""
-    customers = read_json(
-  		json_path: "../customer-themes-tmp/Customers.json"
-	)
-
-	customers.each do |customer|
-		if customer[:customer] == options[:CUSTOMER]
-       		path = "../../customer-themes-tmp/" + customer[:path] + "ios/"
-       		break
-    	end
-	end
-
-	if path == ""
-		UI.user_error!("Customer not found!")
-	end
-
-	sh "cp -R '" + path + "Theming/' ../ownCloud/Resources/Theming/"
-	sh "cp -R '" + path + "AppIcon.appiconset/' ../ownCloud/Resources/Assets.xcassets/AppIcon.appiconset/"
-
-    end
```

Comment:
I don't insist on `applyBranding.sh`. If this makes more sense and we can somehow pass it a path as param, this might be nicer even.